module/programs/helper.js

```diff
@@ -341,6 +344,54 @@ module.exports = class ProgramsHelper {
 					}
 				}
 
+				if (data.components) {
+					let programDocumentRecord = await programsQueries.programsDocument({ _id: programId }, [
+						'components',
+					])
+
+					if (!programDocumentRecord[0]) {
+						throw {
+							message: CONSTANTS.apiResponses.PROGRAM_NOT_UPDATED,
+						}
```

Comment:
@borkarsaish65 if program is not find, why we are sending this message/

Reply:
message changed